node/node.go

```diff
@@ -17,7 +17,7 @@ import (
 )
 
 type StarknetNode interface {
-	Run(ctx context.Context) error
+	Service
```

Comment:
StarknetNode should not be considered a service. Since it is the one responsible for managing all of Juno's services.

Reply:
done